docs/faq.rst

```diff
@@ -4,14 +4,14 @@ Frequently asked question
 Why SQLite3?
 ------------
 
-Although partially answered on the index page, here a more complete answer: If
+Although partially answered on the index page, here is a more complete answer: If
 you manage massive amounts of comments, Isso is a really bad choice. Isso is
-designed to be simple and easy to setup, not optimizied for high-traffic
+designed to be simple and easy to setup, it is not optimizied for high-traffic
 websites (use a `dedicated Disqus`_ instance then).
 
-    comments are not big data
+    Comments are not big data.
 
-For example, 209 threads and 778 comments in total only need 620K (kilobyte)
-memory. Excellent use case for SQLite.
+For example, if you have 209 threads and 778 comments in total this only needs 620 kilobytes 
+of memory. This example would be an excellent use case for SQLite.
```

Comment:
I think the "example" here is superfluous.

Reply:
Should i remove my change or remove the whole example?

Reply:
Perhaps change s/This example would be/This is/